src/app/app-modules/nurse-doctor/radiologist-worklist/radiologist-worklist.component.html

```
@@ -1,227 +1,7 @@
-            [hidden]="dataSource.data.length >= 0"
-            [pageSizeOptions]="[5, 10, 20]"
-            [pageSize]="5"
-            showFirstLastButtons
-            aria-label="Select patient in nurseWorklist">
-          </mat-paginator>
-        </section>
-      </mat-card>
-    </div>
-  </div>
-</div>
+<app-beneficiary-worklist
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)